Libraries/Components/ScrollResponder.js

```diff
@@ -471,6 +472,15 @@ var ScrollResponderMixin = {
    * @platform ios
    */
   scrollResponderFlashScrollIndicators: function() {
+    if (Platform.OS === 'android') {
```

Comment:
Ah; There's some magic inside the `UIManager.js` to handle iOS. Gotcha now :+1:

Updated :)